taskipy/task.py

```diff
@@ -19,6 +20,20 @@ def command(self):
     def description(self):
         return self.__task_description
 
+    @property
+    def use_vars(self):
+        return self.__task_user_vars
+
+    def __extract_task_use_vars(self, task_toml_contents: object) -> bool:
+        if isinstance(task_toml_contents, str):
+            return False
+        if isinstance(task_toml_contents, dict):
+            try:
+                return task_toml_contents['use_vars']
```

Comment:
maybe we can assert that: if the value exist, it's a boolean? and throw `MalformedTaskError` if `use_vars` is not boolean :) (of course, not throw if the value does not exist and just return `False` as you just did ;))